sassoftware/viya4-deployment
# feat: (IAC-862) Update default metrics server helm chart version

### Changes
The viya4-deployment baseline install task deploys the metrics-server component on AWS EKS clusters using a helm chart from bitnami's repository.
The current helm chart version that viya4-deployment uses to install metrics-server on AWS EKS clusters fails to install as it refers to a version that's no longer valid for bitnami's index.yaml file. See issue https://github.com/bitnami/charts/pull/14024 for additional details.
Update the viya4-deployment default helm chart version to use the latest available metrics-server helm chart 6.2.4. Update CONFIG-VARS.md readme file to indicate the new default helm chart version.
### Testing

- Create an AWS cluster created with the latest viya4-iac-aws image registry.unx.sas.com/props/viya4-iac-aws:5.4.0
- Run the viya4-deployment baseline install task for the AWS cluster to verify successful download of the 6.2.4 metrics-server helm chart from bitnami repo
- Verify that the updated metrics-server application deployed successfully
- Verify that the deployed metrics-server is returning data from API endpoint for all current nodes
- Verify a successful Viya deployment with the updated metrics-server installed

See internal ticket for test output details.
